### ISSUE TITLE
Fan control not taking control of Lian Li controller/fans anymore

### ISSUE THREAD

**Issue by NickeyHab** [EightB1ts/FanControl.LianLi#19]
So I just updated L-connect to v2.0.8 because i was trying to see if i could fix some rgb related things. But after doing this it seems that fan control is not taking control of the fans anymore. I use the Uni hub sl-infinity-aio controller with 3 channels, but now when i do assisted setup it says no speed sensor detected even though i can see the rpm in the speeds section.
If i manually set 1 channel to 100% it seems like fan control wants to take over but then after a second it cancels. I tried setting a static rpm in bios with dc, but this didn't help. Setting MB sync on for fans in L-connect only ramps it up to the max, but i cant control it in fan control still. I do have L-connect off all the time usually.
I also have the argb plugin for lian li installed.

**Comment by mminehanNZ** [comment]
I was having the same problem.  Turns out that L-connect runs as service in the background and continues to manage the fans.
Try stopping the L-connect services (two of them) and see if that fixes the probelm, using Task Manager.
Even with L-connect set not to start on startup the services still run. I had to uninstall L-connect completely to stop this from happening.
Not sure if this is your problem but worth a try.

**Comment by marcmaster2009** [comment]
> I was having the same problem. Turns out that L-connect runs as service in the background and continues to manage the fans. Try stopping the L-connect services (two of them) and see if that fixes the probelm, using Task Manager. Even with L-connect set not to start on startup the services still run. I had to uninstall L-connect completely to stop this from happening. Not sure if this is your problem but worth a try.

Are u mange the light effects form the Fan with an other software or over the MB ?

**Comment by mminehanNZ** [comment]
> Are u mange the light effects form the Fan with an other software or over the MB ?

Yes. I'm using SignalRGB with no problems. The fan colour changes are slow though (slow frame rate). That's a limitation of fans though, using USB and the controller.

**Comment by Teku-Guy** [comment]
I have the issue where the fan control is not taking control of the lian li fans properly. I can hard set it to a flat curve after I re auto calibrate the fans. I don't have L-Connect installed or services running I have SignalRgb controlling the less. Lian Li controller is update to the latest lian li software v1.4

**Comment by stubbo66** [comment]
I have two SL-Infinity Controllers, v 1.4, and am using the latest 2.0.13. I was looking for the older 1.6.30 which worked perfectly, but all downloads have been removed from the Lian Li website which is very annoying.

I'm playing whack-a-mole at the moment, as the L-Connect software sees my two hubs and assigns them 1 and 2, but after rebooting with the services disabled they switch around in Fan Control.

It works without the services, but if I every  want to tweak the light configurations it's all a bit of a mess. I use OpenRGB for everything except the Lian Li fans as I prefer their controls so I'm nut using the RGB version of the DLL.

I think most of our issues would be gone if we could go back to that older version of L-Connect 3

**Comment by stubbo66** [comment]
Actually, following on from my last post, the biggest issue I am having is that every time I restart the PC, the controller identities switch over. So all the fans I have configured on controller 1 move to controller two and vice versa. This is an extreme annoyance meaning I have to reconfigure the fans every time I reboot.

Would very much appreciate this bug getting fixed.

**Comment by mminehanNZ** [comment]
I don't think the author of this plugin is active anymore. There has been no activity on his account for a while now.
Make sure you set Lconnect to MB sync and then uninstall it. Basically remove anything to to with the L-connect software including both services.
If you want to use Signal RGB to control the LEDs then you need leave that set to the controller.
Try experimenting with this: https://github.com/EightB1ts/uni-sync
